test/neighborhoods.jl

```diff
@@ -62,4 +62,45 @@
       @test sprint(show, ellipsoid) == "Ellipsoid{Float64}((2.0, 1.0), (45.0,), GSLIB)"
     end
   end
+  
+  @testset "RotationConvention" begin
```

Comment:
I think we can remove the test set and place these tests immediately after the other tests and before the final tests with the `show` methods.